src/goto-instrument/contracts/doc/user/contracts-loops.md

```diff
@@ -5,16 +5,24 @@ Back to @ref contracts-user
 @tableofcontents
 
 CBMC offers support for loop contracts, which includes four basic clauses:
+- an _assigns_ clause for declaring the memory locations assignable by the loop,
 - an _invariant_ clause for establishing safety properties,
 - a _decreases_ clause for establishing termination,
-- an _assigns_ clause for declaring the memory locations assignable by the loop,
 - a _frees_ clause for declaring the pointers freeable by the loop.
+**The four clauses need to be declared in this sequence to avoid errors. Each loop contract should only have one assigns clause that contains all assigned targets.**
```

Comment:
```suggestion

**The three clauses need to be declared in this sequence to avoid errors. Each loop contract should only have one assigns clause that contains all assigned targets.**
```
We don't actually have free clauses in loop contracts.